app/graphql/resolvers/update_user.rb

```diff
@@ -0,0 +1,53 @@
+class Resolvers::UpdateUser < Resolvers::MutationFunction
+  # arguments passed as "args"
+  argument :id, !types.ID
+  argument :first_name, types.String
```

Comment:
I'd argue that first_name, last_name, and email should be required; if you'd like to keep them the same, you can pass the same args. But maybe not.